src/utils/helpers.test.ts

```diff
@@ -37,3 +43,40 @@ describe('abbreviateNumber', () => {
     expect(abbreviateNumber(1119)).toBe('1.12K');
   });
 });
+
```

Comment:
@ajkl2533 should we also test the new ones?

Reply:
We probably can but the question is if it make sense because e.g. `getRadii` is just `path(['radii', type], theme),`.
I added those test to be sure that the changes are backward-compatible